docs/website/docs/building-from-source/getting-started.md

```diff
@@ -109,6 +109,13 @@ cmake --build ../iree-build/
 
 ### Running tests
 
+Build test dependencies:
+
+``` shell
+cmake --build ../iree-build --target iree-test-deps
+# Or `ninja iree-test-deps` in the build directory
```

Comment:
Personally I'd leave this `ninja` suggestion out, just to keep the docs streamlined. I could be convinced to keep it though.